lib/internal/modules/cjs/loader.js

```diff
@@ -1571,6 +1571,11 @@ function getMaybeCachedSource(mod, filename) {
   return content;
 }
 
+/**
+ * Built-in handler for `.cts` files.
+ * @param {Module} module The module to compile
+ * @param {string} filename The file path of the module
+ */
```

Comment:
Can you move this to a separate PR? Let's keep this one focused on the changes in the test file if that's OK. Let me know if you need help, I can push the changes to your branch if you prefer.

Reply:
Sure, no wories

Reply:
Done, @aduh95